### ISSUE TITLE
mxStandardizeRAM allow choices for cov.std?

### ISSUE THREAD

**Issue by tbates** [OpenMx/OpenMx#159]
Is there any utility to adding `type` and `cov.std` options to `mxStandardizeRAM`?

FYI , in lavaan,

`type` for continuous variables
* "std.lv" var of latents only
* "std.all" = var of both manifest and latent variables
* "std.nox"  like "all", but exclude var of exogenous covariates.

`cov.std`
 * If TRUE,  residual observed covariances scaled by sqrt  ‘Theta’ diagonal (residual latent covariances scaled by sqrt ‘Psi’ diagonal).
* If FALSE, sqrt of diagonal of model-implied cov (Sigma), and sqrt diag model-implied cov of latents.

(FYI our default for cov.std is opposite to that of lavaan and M+).

https://rdrr.io/cran/lavaan/man/standardizedSolution.html

**Comment by RMKirkpatrick** [comment]
In my opinion, no.

**Comment by mcneale** [comment]
Does this mean that we should cov.std the same way in the event that someone has specified a type="LISREL" model?  The formulae were in an early version of the LISREL manual, but I vaguely recall that they were withdrawn in later editions.

**Comment by RMKirkpatrick** [comment]
@mcneale `mxStandardizeRAMpaths()` is incompatible with LISREL expectation.

**Comment by mcneale** [comment]
Right, so we could offer an mxStandardizeLISRELpaths() (or add a type='LISREL' option to  mxStandardizeRAMpaths().  In principle, at least...

**Comment by RMKirkpatrick** [comment]
Although not high-priority, implementing `mxStandardizeLISRELpaths()` would be useful.  For such a function, imitating the functionality of `cov.std` would be trivial.  However, it should default to `FALSE`, i.e. by default it should behave as `mxStandardizeRAMpaths()`.  To tell the truth, I find it baffling that lavaan's default is `cov.std=TRUE`.

Per its man page, `mxStandardizeRAMpaths()` defines the standardized solution as " what the path coefficients would be if all variables in the analysis–both manifest and latent–were standardized to zero mean and unit variance".  That's the simplest, sanest, and most sensible definition of "standardized solution" there is.  I don't see any value in adding alternate definitions to `mxStandardizeRAMpaths()`.

**Comment by 0avasns** [comment]
One entirely pragmatic argument in favor of including this option in OpenMx would be to help people transition from Mplus (or helping other senior researchers' students take up R/OpenMx instead of Mplus), because some experienced folks may be a bit nervous switching to something that produces different results (regardless of the justification for the difference).

**Comment by tbates** [comment]
For RAM, it seems like our equivalent of diag Θ/theta is `diag(S[manifests, manifests]`, so we could scale based on that. Does seem like a win.
